openadapt/utils.py

```diff
@@ -26,9 +27,16 @@ def configure_logging(logger, log_level):
     log_level_override = os.getenv("LOG_LEVEL")
     log_level = log_level_override or log_level
     logger.remove()
+    logger_format = ("<green>{time:YYYY-MM-DD HH:mm:ss.SSS}</green> | "
+                     "<level>{level: <8}</level> | "
+                     "<cyan>{name}</cyan>:<cyan>{function}</cyan>:<cyan>{line}</cyan> "
+                     "- <level>{message}</level>")
```

Comment:
Can you please format this as follows:
```
    logger_format = (
        "<green>{time:YYYY-MM-DD HH:mm:ss.SSS}</green> | "
         "<level>{level: <8}</level> | "
         "<cyan>{name}</cyan>:<cyan>{function}</cyan>:<cyan>{line}</cyan> "
         "- <level>{message}</level>"
)
```
Also can you please clarify the origins of this format?